extensions/hp-extension/src/commandsModule.tsx

```diff
@@ -0,0 +1,99 @@
+const commandsModule = ({ servicesManager, commandsManager }) => {
+  const {
+    DisplaySetService,
+    ViewportGridService,
+    HangingProtocolService,
+  } = servicesManager.services;
+
+  const actions = {
+    nextStage: () => {
+      // next stage in hanging protocols
+      HangingProtocolService.nextProtocolStage();
+    },
+    previousStage: () => {
+      HangingProtocolService.previousProtocolStage();
+    },
+
+    updateViewportDisplaySet: ({ direction, circular, sortSource }) => {
+      const {
+        activeViewportIndex,
+        viewports = [],
+      } = ViewportGridService.getState();
+
+      const viewport = viewports[activeViewportIndex];
+      const activeDisplaySetInstanceUIDs = (viewport || {})
+        .displaySetInstanceUIDs;
+
+      if (!activeDisplaySetInstanceUIDs) {
+        throw Error(
+          'Mismatching on ViewportGridService where there is no valid displaySetInstanceUid for activeViewportIndex'
+        );
+      }
+      
+      const activeDisplaySets = DisplaySetService.getActiveDisplaySets();
+      const displaySetGroup = viewport.viewportOptions?.displaySetGroup;
```

Comment:
This line and the next line simply choose to use the default group.  This replaces the default navigate by display set, and has options like wrap etc.